lib/cn/intern_builder.c

```diff
@@ -26,10 +26,20 @@ struct intern_node {
     struct intern_node *next;
 };
 
+/* Key common-prefix-elimination.  If alignment is 4, we build for size
```

Comment:
Build fails!  The alignment ensures the key metadata fields fall on natural work boundaries (esp so that they don't cross a cacheline).  It works transparently on x86 but is suboptimal performance wise (cpu has to do a lot more microps to deal with it)  On many other architectures you'll get some sort of bad address exception if you do that...

Reply:
As long as compilation fails, that is good enough